core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java

```diff
@@ -57,6 +62,8 @@
 
     private static final Logger logger = Logger.getLogger(FileFingerprintStorage.class.getName());
     private static final DateConverter DATE_CONVERTER = new DateConverter();
+    public static final String FINGERPRINTS_DIR_NAME = "fingerprints";
```

Comment:
It is only for tests, right? If the test is applicable to File Fingerprint storage, it makes sense to relocate the dependent test to a new class and use the package scope. Option B is to use `@Restricted(NoExternalUse.class)` to prevent external usages

Reply:
@oleg-nenashev Do I need to add `@Restricted(NoExternalUse.class)` if the entire class itself is using that annotation?